lib/python/ensembl/brc4/runnable/amend_genome_metadata.py

```diff
@@ -0,0 +1,198 @@
+#!env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import re
+from typing import Any, Dict, List, Tuple
+import eHive
+import gzip
+import csv
+import json
+
+from Bio import SeqIO
+
+
+class MissingDataError(Exception):
+    """Used if some data is missing from the report file."""
+    def __init__(self, report_path: str, accession: str, msg):
+        report_msg = f"Can't get data for {accession} in report {report_path}"
+        if msg:
+            report_msg = f"{report_msg}: {msg}"
+        self.msg = report_msg
+
+
+class amend_genome_metadata(eHive.BaseRunnable):
```

Comment:
```suggestion
class AmendGenomeMetadata(eHive.BaseRunnable):
```
This would be PEP8 compliant, but I realise we may not want that since this acts as a runnable... Will leave this decision to you.

Reply:
For now I'll keep it like that (like the other python runnables). We can rename them all at once if we decide to go that way at some point.